### ISSUE TITLE
Homebrew doesn't like installing JS files in "lib"

### ISSUE THREAD

**Issue by jawshooah** [nodenv/node-build-update-defs#9]
```sh
$ brew install nodenv/nodenv/node-build-update-defs
==> Installing node-build-update-defs from nodenv/nodenv
==> Downloading https://github.com/nodenv/node-build-update-defs/archive/v2.0.0.tar.gz
==> Downloading from https://codeload.github.com/nodenv/node-build-update-defs/tar.gz/v2.0.0
######################################################################## 100.0%
🍺  /usr/local/Cellar/node-build-update-defs/2.0.0: 13 files, 17.2K, built in 0 seconds

$ brew audit nodenv/nodenv/node-build-update-defs
nodenv/nodenv/node-build-update-defs:
 * Non-libraries were installed to "/usr/local/Cellar/node-build-update-defs/2.0.0/lib"
Installing non-libraries to "lib" is discouraged.
The offending files are:
  /usr/local/Cellar/node-build-update-defs/2.0.0/lib/build-node.js
  /usr/local/Cellar/node-build-update-defs/2.0.0/lib/build.js
  /usr/local/Cellar/node-build-update-defs/2.0.0/lib/fetch.js
  /usr/local/Cellar/node-build-update-defs/2.0.0/lib/scraper-nodejs.js
  /usr/local/Cellar/node-build-update-defs/2.0.0/lib/scraper.js

Error: 1 problem in 1 formula
```

We could move those files to `libexec`, but that's usually reserved for executables that aren't intended to be run directly by the user. Maybe `src` or `js`?

**Comment by jasonkarns** [comment]
What special thing does homebrew do with that lib dir that makes it so unhappy with js files? And if that behavior is contrary to our use of those script files, is there a way to disable it within the formula? (like a `no_lib` type of thing?) Barring that, I'm probably okay with `src`. But this feels a bit like a homebrew/self-inflicted thing?

**Comment by jasonkarns** [comment]
Doesn't look like there's a way to disable that check: http://www.rubydoc.info/github/Homebrew/brew/FormulaCellarChecks#check_non_libraries-instance_method

**Comment by jawshooah** [comment]
Yeah, it looks like Homebrew automatically symlinks anything in `lib` to `$(brew --prefix)/lib`, so they're picky about what should be included.

**Comment by jasonkarns** [comment]
Ah right. That definitely *should* be something that could be disabled. I'm
sure this can't be the only formula that ought not have lib symlinked (even
if not js). But barring a homebrew pr, switching ours to src is fine.

**Comment by jawshooah** [comment]
An alternative might be to publish this package to npm rather than Homebrew. Makes sense, seeing as it depends on node anyway.

**Comment by jawshooah** [comment]
nodenv/homebrew-nodenv#21 should take care of this without changing anything in this repo.

**Comment by jasonkarns** [comment]
Yeah, I thought about that. But it'd be a bit trickier getting it
registered as nodenv plugin, right?

**Comment by jawshooah** [comment]
Could just do as Homebrew does with `brew` and treat anything in the path prefixed with `nodenv-` as a plugin.

**Comment by jasonkarns** [comment]
resolved by nodenv/homebrew-nodenv#21
